### PR TITLE
User registration fields update

### PR DESCRIPTION
**Pull request by oceanzus**

### CODE REVIEW

**Comment by oceanzus** [comment]
@FBRTMaka @DanielJMaher Please review

**Comment by DanielJMaher** [comment]
@FBRTMaka I reviewed here too,  This looks like what we need.  You good at the front end too?

File: ooiui/templates/common/signup.html [hunk 1]
```diff
@@ -152,12 +190,14 @@
...
     this.listenTo(this, "login:logout", this.onLogout);
+    this.listenTo(this, "country:onchange", this.countrySelection);
   },
   start: function() {
     var self = this;
     this.initEvents();
     this.login.fetch({async:false});
-    var complete = _.invoke([this.collections.roles, this.collections.orgs], 'fetch');
+{#    this.models.stateModel.fetchCurrent('US');#}
+    var complete = _.invoke([this.collections.roles, this.collections.orgs, this.collections.countries, this.collections.states], 'fetch');
```
Comment (FBRTMaka) [inline ooiui/templates/common/signup.html]: @oceanzus nice

**Comment by FBRTMaka** [comment]
@oceanzus, looks sweet! :+1: 

@DanielJMaher all good on my end, back to you!

**Pull request merged**